src/components/Bindings.tsx

```diff
@@ -15,13 +15,8 @@ const Bindings = (props: {
   unbind: (key: string) => void;
   keyMap: KeyMap;
 }): JSX.Element => {
-  const [bindingModalKeys, setBindingModalKeys]: [
-    string,
-    React.Dispatch<React.SetStateAction<string>>
-  ] = useState<string>("");
-  const [bindingModalAction, setBindingModalAction] = useState<Action | null>(
-    null
-  );
+  const [keys, setKeys] = useState<string>("");
+  const [action, setAction] = useState<Action | null>(null);
```

Comment:
@cjquines I split up your commit 5bac5783499759d1b6beefe7b06b977d6ab2258b, and this is one part. I don't like this commit. By looking at this, I don't know what `keys` or `action` are supposed to mean. More, the `keys` parameter (which is aptly named) in a later function ends up shadowing this `keys` variable.

Probably better names are `bindingHotkey` (emphasize that this is one unit, not plural) and `bindingAction` or `hotkeyLabel` and `hotkeyAction`? I also thought they were okay before.

(I get the idea behind not repeating `bindingModal*` within the binding modal component, but this is *not* that component. Removing the explicit typing is great)

Reply:
sure. i strongly dislike the original names, but i think bindingHotkey and hotkeyAction are fine.